### ISSUE TITLE
Prevent-sharing attribute deny dropping items instead of removing them when dropped

### ISSUE THREAD

**Issue by RuedigerLP** [Electroid/PGM#331]
Atm if an item is on prevent-shareing="true" and you drop it it will just get removed, id suggest not removing it but just preventing it from even being dropped. Or maybe having a new attribute that would just be there to deny dropping the item.

**Comment by KingOfSquares** [comment]
So when the player tries to drop the item it would pop back into the inventory?

**Comment by RuedigerLP** [comment]
Yes that would be a possible problem, that should be dealt with on a maddesign level, i guess?

**Comment by Electroid** [comment]
I think from a user's standpoint, this might make sense. It doesn't make sense to accidentally `q` an item and have it disappear. Although this is also an issue with item remove.

Perhaps we need something new?

**Comment by RuedigerLP** [comment]
Yea, for item remove a trick is used combining itemkeep and a clear tag in the spawn kit, so items only disappear upon death but not when dropped by q. Filters for item remove where suggested, see here: https://github.com/Electroid/PGM/issues/112.

**Comment by RuedigerLP** [comment]
In the case of prevent-sharing I'd suggest to allow dropping the item but only allow pickup for the player that dropped it. That would solve item disappearing unwanted and it would also not clog up the inventory.
